README.md

```diff
@@ -85,18 +89,65 @@ const searchClient = instantMeiliSearch(
     paginationTotalHits: 30, // default: 200.
     placeholderSearch: false, // default: true.
     primaryKey: 'id', // default: undefined
+    // ...
   }
 )
 ```
 
-- `placeholderSearch` (`true` by default). Displays documents even when the query is empty.
+### Placeholder Search
+
+Placeholders search means showing results even when the search query is empty. By default it is `true`.
+If set to `false`, only when a request is made with at least one character, results will appear.
```

Comment:
It is maybe came from me but I find this sentence hard to understand

Reply:
```
If set to `false`, results will appear only when a request is made with at least one character.
```

Reply:
> When placeholder search is set to `false`, no results appears when searching on no characters. For example, if the query is `""` no results appear.

Reply:
Perfect